### ISSUE TITLE
Baseline Model Training

### ISSUE THREAD

**Issue by mariafnafees** [mariafnafees/APS360_Project#5]
Selection and reporting of accuracy/loss results from training/validation/testing of Baseline model.
Varying selection of:
- Batch Size
- Iteration number
- Learning Rate
- Optimizer
- Momentum
